fossology/uploads.py

```diff
@@ -368,7 +365,7 @@ def list_uploads(
             for upload in response.json():
                 uploads_list.append(Upload.from_json(upload))
             logger.info(
-                f"Retrieved page {page} of uploads, {response.headers['X-TOTAL-PAGES']} pages are in total available"
+                f"Retrieved page {page} of uploads, {response.headers.get('x-total-pages', 'Unknown')} pages are in total available"
```

Comment:
I just have a question @deveaud-m, shouldn't the header name be `X-Total-Pages`? I am not certain the key match is case-insensitive.

Reply:
> I just have a question @deveaud-m, shouldn't the header name be `X-Total-Pages`? I am not certain the key match is case-insensitive.

None of them will work with the current latest Fossology Docker version anyway... I'd need the API version 1.0.17 to make those tests pass...